app/views/provider/admin/dashboards/widgets/_service_hits.html.slim

```diff
@@ -13,7 +13,7 @@ article.DashboardWidget id = widget.id
           ' In order to show Hits for the #{friendly_service_name(service)} you need to have at least one application sending traffic to it.
         p
           ' Consider
-          => link_to 'making some test calls', edit_admin_service_integration_path(widget_params['service_id'], :anchor => 'staging')
```

Comment:
Where did the `widget_params` come from in the old version? Are you sure you are not overriding anything here?

Reply:
It comes from `widget_params = widget.params.symbolize_keys`

I saw an issue that when accessing service_id as `widget_params['service_id']` because keys were symbolized. That's why I changed that to a symbol.

Anyway, those 2 files are not used and will be deleted by @josemigallas